packages/compass-saved-aggregations-queries/src/stores/open-item.ts

```diff
@@ -259,6 +262,19 @@ export const openSelectedItem =
       return;
     }
 
+    if (updateItemNamespace) {
+      const id = selectedItem.id;
+      const newNamespace = `${selectedDatabase}.${selectedCollection}`;
+
+      if (selectedItem.type === 'aggregation') {
+        await pipelineStorage?.updateAttributes(id, {
+          namespace: newNamespace,
+        });
+      } else if (selectedItem.type === 'query') {
+        await queryStorage?.updateAttributes(id, { _ns: newNamespace });
+      }
```

Comment:
Hmmm, I'd say that as in this case there is no effect on the actual reducer and we already dispatch an action that you can trace back to this handler, it's okay not to do this. Maybe if there was even more logic involved and the state of the update was part of the reducer, this would make more sense to do. Good point though!